### ISSUE TITLE
Windows Python 3.8 asyncio ProactorEventLoop NotImplementedError.

### ISSUE THREAD

**Issue by layabout4233629** [django/daphne#311]
I was leaning use channel in django.
my python enviroment:
python3.8
asgiref==2.3.2
channels==2.1.7
channels-redis==2.3.3
Django==1.11.7
Twisted==19.10.0

when I add app in setting, it always run errors as follow：

Traceback (most recent call last):
  File "C:\Users\ASUS\Desktop\webs\manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Python38\lib\site-packages\django\core\management\__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "C:\Python38\lib\site-packages\django\core\management\__init__.py", line 338, in execute
    django.setup()
  File "C:\Python38\lib\site-packages\django\__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Python38\lib\site-packages\django\apps\registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "C:\Python38\lib\site-packages\django\apps\config.py", line 120, in create
    mod = import_module(mod_path)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\Python38\lib\site-packages\channels\apps.py", line 6, in <module>
    **import daphne.server**
  File "C:\Python38\lib\site-packages\daphne\server.py", line 17, in <module>
    asyncioreactor.install()
  File "C:\Python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 320, in install
    reactor = AsyncioSelectorReactor(eventloop)
  File "C:\Python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 69, in __init__
    super().__init__()
  File "C:\Python38\lib\site-packages\twisted\internet\base.py", line 571, in __init__
    self.installWaker()
  File "C:\Python38\lib\site-packages\twisted\internet\posixbase.py", line 286, in installWaker
    self.addReader(self.waker)
  File "C:\Python38\lib\site-packages\twisted\internet\asyncioreactor.py", line 151, in addReader
    self._asyncioEventloop.add_reader(fd, callWithLogger, reader,
  File "C:\Python38\lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError

**Comment by adamchainz** [comment]
I'm not sure what the problem is, but Django 1.11 does not support Python 3.8 - see  [release notes](https://docs.djangoproject.com/en/3.0/releases/1.11/). I suspect that Twisted version may not support Python 3.8 either. Try using Python 3.7 or even 3.6.

**Comment by carltongibson** [comment]
This is an issue with Windows on 3.8. Will be addressed for Daphne by #298. 

Adam's point about 1.11 holds.

For now you can work around the particular issue setting the even loop policy in your entry point script (`manage.py`): 

```python
import asyncio 

asyncio.set_event_loop_policy(
    asyncio.WindowsSelectorEventLoopPolicy
)
```

**Comment by layabout4233629** [comment]
Thank you very much.
